corehq/apps/hqadmin/management/commands/dump_domain_data.py

```diff
@@ -11,7 +11,9 @@
 app_labels = {
     'locations.LocationType': 'domain',
     'locations.SQLLocation': 'domain',
-    'form_processor.XFormInstanceSQL': 'domain'
+    'form_processor.XFormInstanceSQL': 'domain',
+    'form_processor.XFormAttachmentSQL': 'form__domain',
+    'form_processor.XFormOperationSQL': 'form__domain',
```

Comment:
Will cases be much different, or are you planning to rebuild the cases from the forms?